wp-content/themes/missilethreat/assets/_scss/base/_base.scss

```
@@ -15,26 +15,18 @@ body {
 }
 
 .container {
-  --container-padding: #{rem(20)};
-
-  $padding: (
-    'small': 24,
-    'large': 48,
-    'xlarge': 64,
-  );
+  --container-padding: #{rem(6vw)};
```

```suggestion
  --container-padding: 6vw;
```

Since the value is in `vw`, you don't want to to convert it to rems. :)